Installing-and-Configuring/Upgrading.md

```diff
@@ -81,24 +86,25 @@ the MRC cache, it is necessary to do a second reboot to ensure that
 the TPM values are at their persistent state
 ([issue #150](https://github.com/osresearch/heads/issues/150) aims to fix this).
 
-
 Regenerating the TOTP token
 ---
-![TPM TOTP QR](images/TPM_TOTP_QR.jpg)
+
+![TPM TOTP QR]({{ site.url }}/images/TPM_TOTP_QR.jpg)
 
 After the second post-flash reboot, generate a new token and store the
 QR code in your phone by running:
 
-```
+```shell
```

Comment:
Seeing all of those outdated shell scripts calls makes me extremely unconfortable.
Do not know what to do with that and source of most of the confusion.

Search in page ".sh"

Reply:
Part of the markdownlint is to assign a language to code blocks.  These are not shell scripts as much as they are the shell command in the documentation.

Reply:
My point here was links to old not even existing shell scripts should be removed

Reply:
Oh I see.  I haven't started updating the install documents, so these will be removed or corrected.